mainWindow.py

```diff
@@ -65,37 +66,24 @@ def openFile(self):
         self.table_widget.clear_table()
         
         fileOpener = fileOpenDialog.fileOpenDialog()
+        fileOpener.openFileNameDialog()
+        if fileOpener.fileAName != "":
+            fileOpener.openFileNameDialog()
 
         fileA = fileOpener.fileAName
         fileB = fileOpener.fileBName
-
-        if not os.path.exists(fileA):
-            print(fileA, "Does not exist")
-        if not os.path.exists(fileB):
-            print(fileB, "Does not exist")
-
-        if not utilities.file_readable(fileA):
-            print(fileA + ": Read permission denied.")
-        if not utilities.file_readable(fileB):
-            print(fileB + ": Read permission denied.")
-
-        if not utilities.file_writable(fileA):
-            print(fileA + ": Write permission denied.")
```

Comment:
These printed no matter the input, I removed it because it was simply erroneous terminal printing.